src/modules/attributes.ts

```diff
@@ -26,28 +26,31 @@ function updateAttrs(oldVnode: VNode, vnode: VNode): void {
   if (oldAttrs === attrs) return;
   oldAttrs = oldAttrs || {};
   attrs = attrs || {};
+  const lowAttrs: any = {};
 
   // update modified attributes, add new attributes
   for (key in attrs) {
     cur = attrs[key];
-    old = oldAttrs[key];
+    const lowkey = key.toLowerCase();
```

Comment:
My problem was this test https://github.com/snabbdom/snabbdom/blob/master/test/attributes.js#L65 which uses snabbdom "incorrectly" as you mentioned.